server/src/main/java/org/elasticsearch/action/search/SearchPhaseController.java

```diff
@@ -146,11 +146,12 @@ static SortedTopDocs sortDocs(boolean ignoreFrom, final Collection<TopDocs> topD
         if (topDocs.isEmpty() && reducedCompletionSuggestions.isEmpty()) {
             return SortedTopDocs.EMPTY;
         }
+        // TODO look what we get here
```

Comment:
anything left to be done here?

Reply:
No, that was a bookmark while debugging this. Will remove.